setup.py

```diff
@@ -70,6 +70,7 @@
         "fasteners>=0.16",
         "jinja2>=3.0.3",
         "currentscape>=0.0.11",
+        "importlib_resources>=5.0.0",
```

Comment:
is there a reason we are using importlib_resources instead of Python standard library importlib? Is it because we are using features that are not present in some standard python? Looks like files is there from python 3.9, which is also our minimum version at the moment...

Reply:
Yes, you are right, we can now simply use the one from the standard library. Thanks!

Reply:
Done in the last commit